doc/SEMANTICS.md

```diff
@@ -74,7 +74,7 @@ Mountpoint allows multiple readers to access the same object at the same time.
 However, a new file can only be written to sequentially and by one writer at a time.
 New files that are being written are not available for reading until the writing application closes the file and Mountpoint finishes uploading it to S3.
 If you have multiple Mountpoint mounts for the same bucket, on the same or different hosts, there is no coordination between writes to the same object.
-Your application should not write to the same object from multiple instances at the same time.
+Your application should not write to the same object from multiple instances at the same time. If you use multiple instances for writing the same key (or another client modifies the Object while the file is open for writing), be aware that Mountpoint currently uses Multipart Uploads (MPU) that are created after the file is opened to upload the data to S3.
```

Comment:
Why does Mountpoint using MPU matter? What are the consequences you might need to be aware of?